src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+import { MenuItems } from "./MenuItems";
+import { Button } from "./Button";
+import "./NavStyle.css";
+
+class Navbar extends Component {
+  state = { clicked: false };
+
+  handleClick = () => {
+    this.setState({ clicked: !this.state.clicked });
+  };
+
+  render() {
+    return (
+      <nav className="NavbarItems">
+        <h1 className="navbar-logo">
+          Ducks 'R' Us<i className="fab fa-react"></i>
+        </h1>
+        <div className="menu-icon" onClick={this.handleClick}>
+          <i
+            className={this.state.clicked ? "fas fa-times" : "fas fa-bars"}
+          ></i>
+        </div>
+        <ul className={this.state.clicked ? "nav-menu active" : "nav-menu"}>
+          {MenuItems.map((item, index) => {
+            return (
+              <li key={index}>
+                <a className={item.cName} href={item.url}>
+                  {item.title}
+                </a>
+              </li>
+            );
+          })}
+        </ul>
+        <Button>Cart</Button>
```

Comment:
This button needs to be linked to the cart page

Reply:
Agreed. I was working in the Togglewithit branch and updated the link in that branch.